rpcs3/Emu/Cell/Modules/cellGem.h

```diff
@@ -18,6 +18,7 @@ enum
 	CELL_GEM_ERROR_CONVERT_NOT_STARTED        = 0x80121809,
 	CELL_GEM_ERROR_WRITE_NOT_FINISHED         = 0x8012180A,
 	CELL_GEM_ERROR_NOT_A_HUE                  = 0x8012180B,
+	//CELL_GEM_ERROR_LIGHTING_NOT_CALIBRATED    =;
```

Comment:
?

Reply:
what value does cellGemGetEnvironmentLightingColor return without calibration

Reply:
No such value exists in libgem. Seems like it just returns `CELL_OK` (with rgb to (128, 128, 128)).

Reply:
hmm ok. thx